github/jwt

```diff
@@ -7,7 +7,8 @@ require "openssl"
 require "jwt" # https://rubygems.org/gems/jwt
 
 # Private key contents
-private_pem = File.read(ENV["HOME"] + "/.ssh/github.pem")
+get_secret = File.expand_path("../aws/secretsmanager/get", __dir__)
+private_pem = `#{get_secret} GitHub`
```

Comment:
What happens if `get_secret` fails? Perhaps check `$?.success?` and fail with information on how to debug?